### DISCUSSION TITLE
Does Pulp accept new contributors?

### DISCUSSION THREAD

**Discussion by Ishaanjolly** [coin-or/pulp#777] [General]
Hey, I just meant to ask if Pulp would be keen to have new contributors to look at issues/enhancements? :)

**Comment by pchtsp** [comment]
Yes, for sure. We're interested in improving the documentation of individual solvers, new solver integrations, type hinting across the project and any other issue or improvement from the issue tracker.

**Reply by Ishaanjolly** [reply]
Sure, I noted that the labels are no longer being used for issues: good_first_issue etc are good enough labels for young contributors to `pulp` like me! 

I'd be willing to have a chat, if possible to just help me get started on where I can contribute. I have used `pulp` quite extensively now since studying OR.
